src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -90,10 +90,12 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   }
 
   private var sessionUsed = false
+  private val metricsCategory = s"cassandra.journal.${context.system.name}.${self.path.name}"
 
   private[journal] lazy val cassandraSession: CassandraSession = {
     retry(config.connectionRetries + 1, config.connectionRetryDelay.toMillis) {
       val underlying: Session = clusterBuilder.build().connect()
+      CassandraMetricsRegistry(context.system).addMetrics(metricsCategory, underlying.getCluster.getMetrics.getRegistry)
```

Comment:
It is one of the important features of the library. As far as I know,  It has been supported at least from version 1.7 until 3.0. 
http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/Metrics.html